assets/large_language_models/rag/components/src/validate_deployments.py

```diff
@@ -326,7 +326,12 @@ def validate_aoai_deployments(parser_args, check_completion, check_embeddings, a
                 'apiVersion',
                 connection_metadata.get('ApiVersion', "2023-03-15-preview"))
             completion_params["connection"] = connection_id_completion
-            if connection["properties"]["metadata"].get("ResourceId", None) is not None:
+            # Name is currently the only distinguishing factor between default and non-default
+            # Default connection is the only one which can perform control plane operations,
+            # as AI Studio does not allow selecting of ResourceID in their UI yet.
+            if connection.get("name", None) == "Default_AzureOpenAI":
```

Comment:
nit - this could be a shared function.

def is_default_connection(connection: someType) -> bool:
    return connection.get("name", None) == "Default_AzureOpenAI"

Reply:
Fixed!